src/erlang_ls_protocol_impl.erl

```diff
@@ -124,8 +124,9 @@ textdocument_didchange(Params) ->
   case ContentChanges of
     []                      -> ok;
     [#{<<"text">> := Text}] ->
-      {ok, Buffer} = erlang_ls_buffer_server:get_buffer(Uri),
-      ok = erlang_ls_buffer:set_text(Buffer, Text)
+      {ok, Buffer0} = erlang_ls_db:find(erlang_ls_files, Uri),
```

Comment:
These three calls could be simplified with `erlang_ls_buffer:update(Uri, TexT)` if `erlang_ls_buffer` was the one writing to the table and in general the one that know what table should it use.